include/stl2/detail/algorithm/minmax.hpp

```diff
@@ -24,19 +24,17 @@
 // minmax [alg.min.max]
 //
 STL2_OPEN_NAMESPACE {
-	namespace __minmax {
-		template<InputRange Rng, class Comp = less<>, class Proj = identity>
-		requires
-			Copyable<iter_value_t<iterator_t<Rng>>> &&
-			IndirectStrictWeakOrder<
-				Comp, projected<iterator_t<Rng>, Proj>>
-		constexpr tagged_pair<tag::min(iter_value_t<iterator_t<Rng>>),
-			tag::max(iter_value_t<iterator_t<Rng>>)>
-		impl(Rng&& rng, Comp comp = Comp{}, Proj proj = Proj{})
+	class __minmax_fn {
```

Comment:
Ditto `struct` (I'll stop marking these now - please audit for all occurrences).